### ISSUE TITLE
Ejercicio Tacon

### ISSUE THREAD

**Issue by javierbarbe** [ivanjimenez/EntornosDesarrolloDAW#2]
Buenas Ivan
 para el ejercicio de entornos de la zapateria tacon... para limitar los valores que puede tomar las clases: complementos ( que solamente pueden ser cinturones o guantes ) bolsos ( que solamente pueden ser mochila,fiesta,bandolera)... eso se indica en el uml?  en el esquema de la clase correspondiente ??? gracias

**Comment by ivanjimenez** [comment]
Hola, Javier:

   Eso lo podríamos hacer con un tipo de Datos Enum, aunque no sería necesario en este ejercicio, lo podríamos obviar, sí que sería mejor hacerlo de cara a un diseño más exhaustivo. Como ejemplo imagina que tenemos la Clase ClassA con un tipo de datos DaysOfTheWeek. Las Clase ClassA tendría una relación de asociación unidreccional para DaysOtTheWeek.
```
_______________________
|   <<enumeration>>   |
|    DaysOfTheWeek    |
|_____________________|
| Sunday              |
| Monday              |
| Tuesday             |
| ...                 |
|_____________________|
```

**Comment by javierbarbe** [comment]
vale, muchas gracias ivan
He subido lo que creo que deberia ser el ejercicio ... si puedes echarle un vistazo y me comentas los fallos que veas
un saludo

**Comment by ivanjimenez** [comment]
El ejercicio no es correcto porque debes hacer lo que se solicita paso por paso. Revisa el enunciado:

* Crear la tabla de sustantivos y categorizarlos, seleccionar las clases. 2 puntos.
* Obtener las relaciones. 2 puntos.
* Obtener los atributos. 1 punto.
* Obtener los métodos. 1 punto.
* Crear el diagrama de clases. 3 puntos.
* Añadir documentación. 1 punto

Luego debes entregar tanto en formato UMLlet y png con un documento donde vayas haciendo lo anterior. 

Revisad los ejercicios planteados y solucionados, son de igual procedimiento.

Os he dejado unos días más para entregarlo.

Saludos

**Comment by danieldat** [comment]
Entonces si me he enterado bien,los atributos se ponen en sus clases correspondientes pero no se detalla si es un bolso de fiesta,por ejemplo.

Tipo de campaña,yo he hecho un método que genere campaña y ya el tío que crea la campaña pone la estación de turno.

Constructores getters y setters me da que no se ponen verdad?

Yo el proyecto lo he hecho con el papyrus,ya cuando decidiste cambiar de programa,yo ya lo tenia medio hecho.

Lo de añadir documentación no se a cual se refiere bien,es la parte del punto 5f .8(que pone documentación adicional?)?

Hay que presentar dos documentos uno,con los sustantivos y el otro con el diagrama verdad?
Y los documentos te los podemos presentar en pdf no?

Gracias

**Comment by ivanjimenez** [comment]
Sí, hazlo con Papyrus o Umlet, es igual con el que lo hagas. Los atributos se pone el atributo que te salga en el texto y puedes poner otros si quieres más, pero sólo se pone atributo:tipo_del_atributo. Los getters y setters no se ponen, pero si son dos lo puedes hacer. Lo de documentación es explicar las clases. Revisa los dos ejemplos anteriores y verás que el procedimiento es el mismo. Entregar el diagrama en formato pdf o png además en formato papyrus o umlet. Y luego las anotaciones en otro pdf.
